### ISSUE TITLE
Inaccurate result for `union` examples

### ISSUE THREAD

**Issue by brandonmcconnell** [angus-c/just#502]
Both in the main [README](https://github.com/angus-c/just#just-union) and in the comment within the [linked source](https://github.com/angus-c/just/blob/master/packages/array-union/index.js), the result is presented as `[1, 2, 3, 4, 5, 6]`, but the actual result is `[1, 2, 5, 6, 3, 4]`, since arrays intrinsically respect insertion order.

I found this confusing as it seemed to suggest that union also employed sorting of its own, when in fact it does not, but was rather a misforecasted result.

### TL;DR

Re `union` ([in README](https://github.com/angus-c/just#just-union) | [linked source](https://github.com/angus-c/just/blob/master/packages/array-union/index.js)), `[1, 2, 3, 4, 5, 6]` ≠ `[1, 2, 5, 6, 3, 4]`

**Comment by brandonmcconnell** [comment]
This issue will be fixed by merging PR #503
